tests/storage_test.go

```diff
@@ -141,6 +141,7 @@ var _ = Describe("Storage", func() {
 
 				By("Checking that /dev/vdc has a capacity of 2Gi")
 				res, err := expecter.ExpectBatch([]expect.Batcher{
+					&expect.BSnd{S: "\n"},
```

Comment:
I think that the test here already use `tests.LoggedInAlpineExpecter(vmi)`, that should already take care of the `\n`.

Reply:
ok, actually the `\n` should also not be needed here, since we directly send something ...